flycheck.el

```diff
@@ -12255,7 +12255,7 @@ information about tflint."
 See URL `https://github.com/terraform-linters/tflint'."
   :command ("tflint" "--format=json"
             (option-list "--var-file=" flycheck-tflint-variable-files concat)
-            source-original)
+            (option-list "--chdir=" source-directory concat))
```

Comment:
Do you think it make sense to apply `--filter` as well.

See https://github.com/terraform-linters/tflint/pull/1687
```
$ tflint --chdir=modules/production --filter=main.tf
// replacement of "tflint modules/production/main.tf"
```

Reply:
Good point.